src/mixins/namex-api-mixin.ts

```diff
@@ -0,0 +1,99 @@
+// Libraries
+import axios from '@/utils/axios-auth'
+import { Component, Vue } from 'vue-property-decorator'
+import { NameRequestStates } from '@/enums/nameRequestStates'
+
+/**
+ * Temporary mixin to help with name request calls until we get the legal api proxy
+ */
+@Component
+export default class NameXApiMixin extends Vue {
+  // eslint-disable-next-line
+  private token: string;
+  private tokenUrl: string = 'https://sso-dev.pathfinder.gov.bc.ca/auth/realms/sbc/protocol/openid-connect/token'
+  private accountId: string = 'lear_sa'
+  private accountSecret: string = '72a733b4-4677-41be-8e66-d9ccaaafa60a' // Input secret for development
+
+  /**
+   * Temporary method for initializing a namex api token for development
+   * This will be removed in the future when supporting dependencies are in place
+   */
+  async intializeNameXToken () {
```

Comment:
If you want to add a return type here i imagine it might be something like `: Promise<any>`

Reply:
updated